docs/versioned_docs/version-zenith/guides/snippets/github-google-cloud/typescript/index.ts

```diff
@@ -0,0 +1,40 @@
+import { dag, Container, Directory, Secret, object, func } from "@dagger.io/dagger"
+
+@object()
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class MyModule {
+
+  source: Directory
+
+  // constructor
+  constructor (source: Directory) {
+    this.source = source
+  }
+
+  @func()
+  build(): Container {
+    return dag.container().from("node:21")
+      .withDirectory("/src", this.source)
+      .withWorkdir("/src")
+      .withExec(["cp", "-R", ".", "/home/node"])
```

Comment:
This one is still doing the manual copy.

Reply:
Fixed